minigrid/wrappers.py

```diff
@@ -127,16 +127,15 @@ def reset(self, **kwargs):
         return self.env.reset(**kwargs)
 
 
-# Should be named PositionBonus
-class StateBonus(Wrapper):
+class PositionBonus(Wrapper):
```

Comment:
Could you add a note that this was previously called StateBonus

Reply:
I added it into the docstring.